webview-ui/src/components/chat/ChatTextArea.tsx

```diff
@@ -508,6 +510,42 @@ const ChatTextArea = forwardRef<HTMLTextAreaElement, ChatTextAreaProps>(
 			setIsMouseDownOnMenu(true)
 		}, [])
 
+		// Utility function to estimate text width
```

Comment:
Is there a way to get the actual width of the selected text? I'm just a little nervous about the estimates.

Reply:
Ah I think we can, let me have a look.

Reply:
Got it as accurate as it can be 🤔